appserver/tests/quicklook/build.xml

```diff
@@ -216,7 +216,7 @@
         <ant dir="admincli" target="build"/>
         <ant dir="adminconsole" target="build"/>
         <ant dir="rest" target="build"/>
-        <ant dir="osgitest/basicosgi" target="build"/>
+        <!--<ant dir="osgitest/basicosgi" target="build"/>-->
```

Comment:
Test skipped as FighetFish component not distributed currently. See: https://github.com/eclipse-ee4j/glassfish/issues/22994 https://github.com/eclipse-ee4j/glassfish/pull/22993